package.json

```diff
@@ -56,6 +56,7 @@
     "gatsby-transformer-remark": "^6.14.0",
     "gatsby-transformer-sharp": "^5.14.0",
     "husky": "^7.0.4",
+    "lodash": "^4.17.21",
```

Comment:
This should always have been here. Aparently we were relying on a transitive dep.